docs/install-and-use.md

```diff
@@ -93,6 +93,26 @@ If your application does use Slimmer/Static:
 //= require govuk_publishing_components/components/button
 ```
 
+As of v5.0.0, the target for browser support for govuk-frontend has changed to browsers that support using the `type="module"` attribute on `script` tags. This means that that govuk_publishing_components Javascript files that include files govuk-frontend will now error ungracefully and potentially function in an unpredictable way if a browser that doesn't support `type="module"` tries to evaluate the Javascript in a script tag without the `type="module"` attribute.
```

Comment:
`This means that that govuk_publishing_components Javascript files that include files govuk-frontend`

Should this be `that include files from govuk-frontend`?

Reply:
Good spot, this is now fixed 👍